DataProcessing/datax-keyvault/pom.xml

```diff
@@ -2,251 +2,217 @@
-            <arg>-deprecation</arg>
-            <arg>-feature</arg>
-          </args>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.10.1</version>
-        <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <version>3.3.0</version>
-        <executions>
-          <execution>
-            <id>copy-dependencies</id>
-            <phase>package</phase>
-            <goals>
-              <goal>copy-dependencies</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${project.build.directory}/dependency</outputDirectory>
-              <overWriteReleases>false</overWriteReleases>
-              <overWriteSnapshots>true</overWriteSnapshots>
-              <excludeTransitive>true</excludeTransitive>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <!-- enable scalatest -->
-      <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <version>3.4.1</version>
-        <configuration>
-          <descriptors>
-            <descriptor>with-dependencies.xml</descriptor>
-          </descriptors>
-        </configuration>
-        <executions>
-          <execution>
-            <id>make-assembly</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-          <plugin>
-            <groupId>com.github.github</groupId>
-            <artifactId>site-maven-plugin</artifactId>
-            <version>0.12</version>
-            <configuration>
-              <message>Maven artifacts for ${project.version}</message>
-              <noJekyll>true</noJekyll>
-              <outputDirectory>${project.build.directory}/mvn-repo</outputDirectory>
-              <branch>refs/heads/mvn-repo</branch>
-              <includes>
-                <include>**/*</include>
-              </includes>
-              <repositoryName>data-accelerator</repositoryName>
-              <repositoryOwner>Microsoft</repositoryOwner>
-              <merge>true</merge>
-            </configuration>
-            <executions>
-              <!-- run site-maven-plugin's 'site' target as part of the build's normal 'deploy' phase -->
-              <execution>
-                <goals>
-                  <goal>site</goal>
-                </goals>
-                <phase>deploy</phase>
-              </execution>
-            </executions>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-    <profile>
-      <id>publish</id>
-      <build>
         <plugins>
-          <plugin>
-            <artifactId>maven-deploy-plugin</artifactId>
```

Comment:
Same here. does deployment work without this ?

Reply:
recovered, the deploy plugin should be kept